test/e2e/main_test.go

```diff
@@ -24,16 +24,18 @@ import (
 )
 
 var (
-	f                  = framework.Global
-	t                  *testing.T
-	namespace          string
-	nodes              []string
-	startTime          time.Time
-	bond1              string
-	bridge1            string
-	primaryNic         = flag.String("primaryNic", "eth0", "Primary network interface name")
-	firstSecondaryNic  = flag.String("firstSecondaryNic", "eth1", "First secondary network interface name")
-	secondSecondaryNic = flag.String("secondSecondaryNic", "eth2", "Second secondary network interface name")
+	f                    = framework.Global
+	t                    *testing.T
+	namespace            string
+	nodes                []string
+	startTime            time.Time
+	bond1                string
+	bridge1              string
+	primaryNic           = flag.String("primaryNic", "eth0", "Primary network interface name")
+	firstSecondaryNic    = flag.String("firstSecondaryNic", "eth1", "First secondary network interface name")
+	secondSecondaryNic   = flag.String("secondSecondaryNic", "eth2", "Second secondary network interface name")
+	nodesInterfacesState = make(map[string][]byte)
```

Comment:
What about adding a "previous" prefix or "expected" prefix to nodeInterfacesState ?